public/views/sponsors.html

```diff
@@ -52,14 +52,18 @@
         <div class="container">
             <div class="row">
                 <div class="col-sm-12 my-title">
-                    <h>TITLE</h>
+                    <h>Interested in Becoming a Sponsor?</h>
```

Comment:
For design consistency, all titles should be in all-caps. However then this text becomes too long. How about "WANT TO BECOME A SPONSOR?"